From d9957ca0a0031e843b8be61a54a8e23fea94427f Mon Sep 17 00:00:00 2001
From: "yadi.hu@windriver.com" <yadi.hu@windriver.com>
Date: Thu, 13 Nov 2014 14:22:52 +0800
Subject: [PATCH 2/4] use/serial:Fix a warning: 'return' with a value, in
 function returning void

linux/drivers/usb/serial/usb-serial.c: In function 'serial_poll_put_char':
linux/drivers/usb/serial/usb-serial.c:1286:3: warning: 'return' with a value, in function returning void

Signed-off-by: Hu <yadi.hu@windriver.com>
---
 drivers/usb/serial/usb-serial.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/usb/serial/usb-serial.c b/drivers/usb/serial/usb-serial.c
index 1d5205f..950ca4c 100644
--- a/drivers/usb/serial/usb-serial.c
+++ b/drivers/usb/serial/usb-serial.c
@@ -1283,7 +1283,7 @@ static void serial_poll_put_char(struct tty_driver *driver, int line, char ch)
 
 	port = usb_serial_port_get_by_minor(line);
 	if (!port)
-		return -1;
+		return;
 
 	serial = port->serial;
 	if (!serial)
-- 
1.7.5.4

